### PR TITLE
Added eu-indicators via new fork

### PR DESCRIPTION
**Pull request by tuopouk**
@Coding-with-Adam Here's new commit from a new fork. Please check if this works.

### CODE REVIEW

**Comment by Coding-with-Adam** [comment]
Thank you @tuopouk 
Now this app can be merged with no conflicts. I appreciate you re-forking the repo. 

This app is a nice example of a clientside callback and a chained callback.

@AnnMarieW do you have any suggestions for this app to simplify it?

File: examples/eu-indicators.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+}
+# Convert the data to a pandas dataframe
+bytes_data = requests.get(file_url, headers=headers).content
+s = str(bytes_data, "utf-8")
+data = StringIO(s)
+df = pd.read_csv(data, sep=";", header=None)
+df = df.set_index(list(df.columns[:-2])).dropna(axis=0, how="all").reset_index()
+df.drop([1, 3, 7], axis=1, inplace=True)
+df.columns = ["Indicator", "Country", "Gender", "Year", "Val"]
+df.Val = df.Val.str.replace(",", ".").astype(float)
```
Comment (Coding-with-Adam) [inline examples/eu-indicators.py:25]:
```suggestion
df = pd.read_csv('https://raw.githubusercontent.com/plotly/datasets/master/Dash-Examle-Index/eu-indicators.csv')
```

File: examples/eu-indicators.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
+import pandas as pd
+import requests
+from dash import Dash, dcc, html, Input, Output
+import dash_bootstrap_components as dbc
+import plotly.express as px
+from io import StringIO
+
+# This a link to the web interface where the data can be downloaded.
+web_url = "https://sotkanet.fi/sotkanet/en/taulukko/?indicator=LclBCoAwEAPAF6WwWymFnMVnBBXrST0oCL7eFbyESUI_D00XsRCbyj0QVmX0hEJU5mTKhMdklVjjVrPIrJ0Yv-odkeVBoqmffxFP4AU=&region=s05zsy7TM4w3sjYCkvnWqXqGAA==&year=sy5ztk7V0zUEAA==&gender=m;f;t&abs=f&color=f&buildVersion=3.1.1&buildTimestamp=202211091024"
+df = pd.read_csv('https://raw.githubusercontent.com/plotly/datasets/master/Dash-Examle-Index/eu-indicators.csv')
```
Comment (Coding-with-Adam) [inline examples/eu-indicators.py:10]:
```suggestion

df = pd.read_csv('https://raw.githubusercontent.com/plotly/datasets/master/Dash-Examle-Index/eu-indicators.csv')

```

File: examples/eu-indicators.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                        ),
+                    ]
+                )
+            ],
+            className="text-center",
+        ),
+        html.Div(id="eu-indicators-x-printing-hidden-content"),
+    ]
+)
+app.clientside_callback(
```
Comment (Coding-with-Adam) [inline examples/eu-indicators.py:76]:
```suggestion
dash.clientside_callback(
```

File: examples/eu-indicators.py [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+        fig = px.line(dff, x="Year", y="Val", color="Gender", height=500)
+        fig.update_traces(mode="lines+markers", line=dict(width=3))
+    fig.update_layout(
+        title=dict(text=f"{indicator} in {country}<br>{title_text} by gender", x=0.5),
+        yaxis=dict(title=indicator),
+        xaxis=dict(title="Years"),
+    )
+    return dcc.Graph(figure=fig)
+
+@app.callback(
```
Comment (Coding-with-Adam) [inline examples/eu-indicators.py:121]:
```suggestion
@dash.callback(
```

File: examples/eu-indicators.py [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+          window.print()
+        }
+        return ""
+    }
+    """,
+    Output("eu-indicators-x-printing-hidden-content", "children"),
+    Input("eu-indicators-x-printing", "n_clicks"),
+)
+
+@app.callback(
```
Comment (Coding-with-Adam) [inline examples/eu-indicators.py:89]:
```suggestion
@dash.callback(
```

File: examples/eu-indicators.py [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+                        ),
+                    ]
+                )
+            ],
+            className="text-center",
+        ),
+        html.Div(id="eu-indicators-x-printing-hidden-content"),
+    ]
+)
+dash.clientside_callback(
```
Comment (Coding-with-Adam) [inline examples/eu-indicators.py:76]:
```suggestion
@dash.clientside_callback(

```

File: examples/eu-indicators.py [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+          window.print()
+        }
+        return ""
+    }
+    """,
+    Output("eu-indicators-x-printing-hidden-content", "children"),
+    Input("eu-indicators-x-printing", "n_clicks"),
+)
+
+@dash.callback(
```
Comment (Coding-with-Adam) [inline examples/eu-indicators.py:90]:
```suggestion
@app.callback(
```

File: examples/eu-indicators.py [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+        fig = px.line(dff, x="Year", y="Val", color="Gender", height=500)
+        fig.update_traces(mode="lines+markers", line=dict(width=3))
+    fig.update_layout(
+        title=dict(text=f"{indicator} in {country}<br>{title_text} by gender", x=0.5),
+        yaxis=dict(title=indicator),
+        xaxis=dict(title="Years"),
+    )
+    return dcc.Graph(figure=fig)
+
+@dash.callback(
```
Comment (Coding-with-Adam) [inline examples/eu-indicators.py:122]:
```suggestion
@app.callback(
```

File: examples/eu-indicators.py [hunk 9]
```diff
@@ -0,0 +1,138 @@
+import pandas as pd
+import requests
+from dash import Dash, dcc, html, Input, Output
```
Comment (Coding-with-Adam) [inline examples/eu-indicators.py:3]:
```suggestion
from dash import Dash, dcc, html, Input, Output, clientside_callback
```

File: examples/eu-indicators.py [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+                        ),
+                    ]
+                )
+            ],
+            className="text-center",
+        ),
+        html.Div(id="eu-indicators-x-printing-hidden-content"),
+    ]
+)
+@dash.clientside_callback(
```
Comment (Coding-with-Adam) [inline examples/eu-indicators.py:76]:
```suggestion
clientside_callback(
```

File: examples/eu-indicators.py [hunk 11]
```diff
@@ -0,0 +1,138 @@
+import pandas as pd
+import requests
+from dash import Dash, dcc, html, Input, Output, clientside_callback
+import dash_bootstrap_components as dbc
+import plotly.express as px
+from io import StringIO
```
Comment (Coding-with-Adam) [inline examples/eu-indicators.py:6]:
```suggestion
```

**Comment by Coding-with-Adam** [comment]
@tuopouk 
Don't mind the numerous commits. I made a small mistake updating all the callbacks when I really just wanted to update clientside_callback, based on Ann's suggestions to me. (it needs to be `dash.clientside_callback` rather than `app.clientside_callback`, just like you did with your Finland app.)

I also created a csv file from the download link and put it on the plotly dataset repo. I like your headers example, showing people how to read a file from a url, but I'm worried that the download link won't work in the future and the app will break. So for longevity purposes I put the file on Plotly's github.

Ann suggested we do a forum post to show the original version and explain how to get data like that into a dash app.  Then we could link to that post in the Example App Index. I like that idea and will try to implement this later today.

**Comment by tuopouk** [comment]
@Coding-with-Adam 
Sorry, I thought I replied it's ok for a while ago.  😆 
Yeah, it's ok, if you can merge still 👍

**Comment by AnnMarieW** [comment]
@tuopouk 
I know this is ancient history, but I was just doing some clean-up and found this app.   I totally missed it previously -- I'm so sorry!  I thought it was a duplicate of one that was merged previously:  https://dash-example-index.herokuapp.com/key-figures-finland

I'm merging this now and it will be live with the next batch of apps in the next week or two.

Nice app!  and thanks so much for this contribution.

**Comment by tuopouk** [comment]
@AnnMarieW No worries. Thanks for replying, I appreciate it!

**Pull request merged**